components/auth/RegisterSignIn.tsx

```diff
@@ -92,23 +93,27 @@ const SignIn: FC<TProps> = ({ userName, onChangePageState }) => {
 
     if (validate()) {
       setLoading(true);
-
       // call api
-      const userInfo = {
-        name: userName,
-        account,
-        password,
-      };
-      alert(JSON.stringify(userInfo));
-
+      const [err, issuer] = await signIn({
+        userInfo: {
+          email: account,
+          password,
+        },
+      });
+      if (!issuer) {
+        // if error
+        // console.log('err', err)
+        setError(err);
+        setLoading(false);
+        return false
+      } else {
       // if success
+      // console.log('issuer',issuer)
       onChangePageState(RegisterPageState.FinishPage);
```

Comment:
Do we need to update user info from server into context  before we change page ?